server/src/modules/auth/auth.module.ts

```diff
@@ -65,6 +66,7 @@ import { SessionScheduler } from 'src/schedulers/session.scheduler';
       },
       inject: [ConfigService],
     }),
+    TooljetDbModule,
```

Comment:
It is due to the dependency of `TooljetDbOperationsService` required in `PluginsHelper`
If I remove it I get the error:
```sh
Error: Nest can't resolve dependencies of the PluginsHelper (PluginRepository, ?). Please make sure that the argument TooljetDbOperationsService at index [1] is available in the AuthModule context.

Potential solutions:
- If TooljetDbOperationsService is a provider, is it part of the current AuthModule?
- If TooljetDbOperationsService is exported from a separate @Module, is that module imported within AuthModule?
```

Ideally, we should remove plugins helper from auth module. But since such refactors will go beyond scope I rather started adding `TooljetDbModule` as a dependency everywhere such that when a refactor happens we can simply delete it from all the irrelevant modules.